### ISSUE TITLE
KeyError on givenName when logging in to /ownership with service account

### ISSUE THREAD

**Issue by dleehr** [Duke-GCB/D4S2#59]
```
Environment:


Request Method: GET
Request URL: https://d4s2.gcb.duke.edu/

Django Version: 1.9.4
Python Version: 2.7.12
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'rest_framework',
 'd4s2_api',
 'switchboard',
 'crispy_forms',
 'ownership',
 'simple_history',
 'shibboleth']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'simple_history.middleware.HistoryRequestMiddleware',
 'shibboleth.middleware.ShibbolethRemoteUserMiddleware']



Traceback:

File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  123.                 response = middleware_method(request)

File "/usr/src/app/src/django-shibboleth-adapter/shibboleth/middleware.py" in process_request
  42.         user = auth.authenticate(remote_user=username,meta=request.META)

File "/usr/local/lib/python2.7/site-packages/django/contrib/auth/__init__.py" in authenticate
  74.             user = backend.authenticate(**credentials)

File "/usr/src/app/src/django-shibboleth-adapter/shibboleth/backends.py" in authenticate
  38.                 user = self.configure_user(user,meta)

File "/usr/src/app/src/django-shibboleth-adapter/shibboleth/backends.py" in configure_user
  49.                 user.__setattr__(attribute["user_attribute"] ,meta[attribute["shibboleth_key"]])

Exception Type: KeyError at /
Exception Value: 'givenName'
```

**Comment by dleehr** [comment]
This exception is raised because the givenName key is marked required in [django-shibboleth-adapter](https://github.com/Duke-GCB/django-shibboleth-adapter/blob/3de247a9a9f726a3f8cf054d58b8cdb6295eb700/shibboleth/backends.py#L51), but is not present

**Comment by dleehr** [comment]
Fixes with #62
